app/views/analytics/districts/show.csv.erb

```diff
@@ -1,10 +1,8 @@
-<%= [
-  'Report generated at:',
-  Time.current
-].to_csv -%>
+<% csv = CSV.generate(headers: true) do |csv| %>
```

Comment:
This generates a CSV and calls `.html_safe` on the final output instead of doing `.to_csv.html_safe` on all the individual strings.